CMakeLists.txt

```diff
@@ -238,6 +238,15 @@ list(APPEND libs ${thirdparty_lib})
 
 add_subdirectory(library)
 
+if(ENABLE_TESTING OR ENABLE_PROGRAMS)
+    file(GLOB MBEDTLS_TEST_FILES ${CMAKE_CURRENT_SOURCE_DIR}/tests/src/*.c)
+    add_library(mbedtls_test OBJECT ${MBEDTLS_TEST_FILES})
+    target_include_directories(mbedtls_test
+        PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/tests/include
+        PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/include
```

Comment:
This looks harmless, but redundant. Sample programs already had the toplevel `include` directory in their include path.

Reply:
This is specific to the `mbedtls_test` target thus test common code and independent of the include directories for programs.